tortoisebeacon/proposal_list.go

```diff
@@ -11,8 +11,7 @@ import (
 
 type proposalList []proposal
 
-// Sort sorts proposal list.
-func (hl proposalList) Sort() []proposal {
+func (hl proposalList) sort() []proposal {
```

Comment:
unrelated to pr, but isn't this copy unnecessary?

Reply:
It can be removed